rdbi-recipes/src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/MultiChannelSchedulerDAO.java

```diff
@@ -284,8 +288,14 @@ int deleteJobFromReady(
             "     redis.call('SREM', $multiChannelSet$, $channelPrefix$)\n" +
             "  end\n" +
             "end\n" +
-            "if removed > 0 then\n" +
-            "  redis.call('DECRBY', $runningCount$, removed)\n" +
+            "if removedFromRunning > 0 then\n" +
```

Comment:
do you need to do something like 
` if removedFromRunning and tonumber(removedFromRunning)` here as well?

Reply:
no, the zrem call should always return the number of entries removed, where as the if thing and tonumber(thing) is needed for keys that may not exist